src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetLicenceConditionService.kt

```diff
@@ -21,8 +21,13 @@ class GetLicenceConditionService(
     if (crn != null) {
       licences = createAndVaryLicenceGateway.getLicenceSummaries(id = crn)
       licences.data.forEach {
-        val conditions = createAndVaryLicenceGateway.getLicenceConditions(it.id)
-        it.conditions = conditions.data
+        val licenceId = it.id.toIntOrNull()
+        if (licenceId != null) {
+          val conditions = createAndVaryLicenceGateway.getLicenceConditions(licenceId)
+          it.conditions = conditions.data
+        } else {
+          it.conditions = emptyList()
+        }
```

Comment:
Possible refactor for readability if you wanted:

`val licences = createAndVaryLicenceGateway.getLicenceSummaries(id = crn)
licences.data.forEach { summary ->
    summary.conditions = summary.id.toIntOrNull()?.let { licenceId ->
        createAndVaryLicenceGateway.getLicenceConditions(licenceId).data
    } ?: emptyList()
}
`